src/remage-from-lh5.cc

```diff
@@ -28,19 +28,31 @@ int main(int argc, char** argv) {
   bool verbosity = false;
   bool dry_run = false;
   std::vector<std::string> file_names;
+  std::set<std::string> aux_ntuples = {}; // not implemented
```

Comment:
probably not needed for now; remage-from-lh5 is only used for input files for the vertex reader. I do not ethink that we need this for now there.